### ISSUE TITLE
Code preview is broken in Firefox browser

### ISSUE THREAD

**Issue by Liubov-crypto** [aeternity/ae-token-minter#7]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x ] bug report
* [ ] feature request
* [ ] question about the decisions made in the repository

### Issue description

**Current behavior:**
Code preview is broken in Firefox browser when I'm minting token. 

**Expected behavior:**
Should be visible. 

**Steps to reproduce:**
1. Open minter in FF browser 
2. Click on Tokens
3. Check Code preview

![ff](https://user-images.githubusercontent.com/69896204/218142252-a8783e02-fe00-46f8-aead-702542463afd.png)

![cons](https://user-images.githubusercontent.com/69896204/218142745-d1b3407c-a698-4f15-9b66-308d1fa81479.png)

**Comment by nikita-fuchs** [comment]
Yeah, this one cloudflare f***d up. The editor library is loaded from the unpkg CDN from cloudflare, and the performance is abysmal for some reason ! If you reload a few times it shows up. I need to bundle the scripts internally to fix this mess.
